docs/_layouts/_subpage.html

```diff
@@ -0,0 +1,39 @@
+{% extends "_layouts/_base.html" %}
+
+{% block body %}
+<div class="content clearfix">
+	<aside class="col-sm-3">
+		<div class="pages">
+			<p><strong>Documentation</strong></p>
+			<ul>
+				<li><a href="getting-started.html">Getting Started</a></li>
+				<li><a href="templating.html">Templating</a></li>
+				<li><a href="api.html">API</a></li>
+				<li><a href="faq.html">FAQ</a></li>
+			</ul>
+		</div>
+		<div class="toc">
+			<p><strong>{{ page.title }}</strong></p>
+			<ul>
+				{% for heading in toc %}
+					<li>
```

Comment:
Suggestion: use `heading.level` as part of CSS class to nest only visually:
```jinja
<li class="toc-level-{{ item.level }}">
  <a href="#{{ item.id }}">{{ item.html | safe }}</a>
</li>
```
used here: https://github.com/jbmoelker/voorhoede-page-formatter/blob/master/src/templates/project.html#L29